### PR TITLE
Bugfix/ds 458 listing component fix auto escaping issues

### PR DESCRIPTION
**Pull request by adamszalapski**
## Jira

https://pegadigitalit.atlassian.net/browse/DS-458

## Summary

Fix auto escaping after integration to Drupal

## Details

We handle concatenation in a different way that is better handled in Drupal.

## How to test

It should be tested on the Drupal website.

### CODE REVIEW

**Review by mikemai2awesome — APPROVED** [review]

File: packages/components/bolt-listing-teaser/src/listing-teaser.twig [hunk 1]
```diff
@@ -54,11 +54,11 @@
               name: 'chevron-right',
             } only %}
           {% endset %}
-          {% set _read_more_link %}
-            {{ 'Read More'|t }}{{ _icon_chevron_right|spaceless }}
-          {% endset %}
+          {% set _more_info_content %}
+            {{ this.data.more_info.value }} &hellip; {{ 'Read More'|t }}{{ _icon_chevron_right }}
```
Comment (remydenton) [inline packages/components/bolt-listing-teaser/src/listing-teaser.twig:58]: Any reason you didn't include `|spaceless` after `_icon_chevron_right`?  I think that shouldn't cause autoescaping issues, though I'm not entirely sure why it's necessary.  @mikemai2awesome?

**Review by remydenton — APPROVED** [review]

File: packages/components/bolt-listing-teaser/src/listing-teaser.twig (hunk 1 above)
Comment (adamszalapski) [reply packages/components/bolt-listing-teaser/src/listing-teaser.twig:58]: In my opinion, spaceless removes spaces, and I don't see any point to use spaceless for icon and read more if we rendered them without any space, but if you guys thinks that I should add it I can do that.

Comment (mikemai2awesome) [reply packages/components/bolt-listing-teaser/src/listing-teaser.twig:58]: I double checked this and tried to put spaceless in different places, didn't seem to get it to work. I didn't want to hold this up, but keep in mind the chevron can be pushed to the next line if we don't use spaceless due to a white space between text and icon. Where can we put spaceless to make this work? @remydenton @adamszalapski

**Review by danielamorse — APPROVED** [review]

File: packages/components/bolt-listing-teaser/src/listing-teaser.twig (hunk 1 above)
Comment (remydenton) [reply packages/components/bolt-listing-teaser/src/listing-teaser.twig:58]: As discussed with @mikemai2awesome on Slack, we still have some widowing to take care of, but it's a problem on master right now anyway, so let's merge this and @mikemai2awesome will figure it out later 😄 .

**Pull request merged**